### ISSUE TITLE
review search feature

### ISSUE THREAD

**Issue by salus-sage** [janastu/blog.janastu.org#3]

**Comment by lafeo** [comment]
hey is anyone working on this? is the search feature being implemented?

**Comment by tbdinesh** [comment]
No. I think. Esp, indexing the many content repos. Open, Blog, crafts, anthill, iruway ... And some bitly short links which point to  hackmd and gdocs..

**Comment by lafeo** [comment]
alright, ill do this. if possible can you outline the extent like according to what ive understood so far is the following - 
1. index all content repos including the short urls that point to external sources such as hackmd and gdocs
2. make all the above searchable by keyword?

the search feature itself doesnt seem to work right?

**Comment by tbdinesh** [comment]
The current search is only available on open.janastu.org and it does not
index other sites i mentioned
